snapcraft/internal/lifecycle.py

```diff
@@ -326,8 +326,12 @@ def _tar_filter(tarinfo):
 
 def containerbuild(step, project_options, output=None, args=[]):
     config = snapcraft.internal.load_config(project_options)
+    remote = os.environ.get('SNAPCRAFT_CONTAINER_BUILDS')
+    if remote in ['1', '0', None, '']:
+        remote = None
```

Comment:
Note that I'm specifically sticking to the environment variable here to avoid exposing it in the CLI - we might consider to at some point use the --remote argument for all commands that can be run in a container.